api/v1alpha1/clusterrequest_types.go

```diff
@@ -66,6 +66,8 @@ type NodePoolRef struct {
 	Name string `json:"name,omitempty"`
 	// Contains the namespace of the created NodePool.
 	Namespace string `json:"namespace,omitempty"`
+	// Represents the timestamp of the first status check for hardware provisioning
+	StatusCheckStart metav1.Time `json:"statusCheckStart,omitempty"`
```

Comment:
StatusCheckStart seems generic, what about HardwareProvisioningStart?

Reply:
I am adding `ClusterProvisionStartedAt metav1.Time` in the `ClusterDetails`

Reply:
I will rename it.

Reply:
It has been renamed to HardwareProvisioningCheckStart since it tracks the timestamp of the first status check for hardware provisioning.